.travis.yml

```diff
@@ -2,3 +2,5 @@ language: node_js
 cache: yarn
 node_js:
   - node
+before_install:
+  - npm install -g yarn
```

Comment:
Yarn is already installed in Travis build machine

Reply:
Yep I removed that

Reply:
done